cmd/mobile-security-service/main.go

```diff
@@ -57,12 +62,15 @@ func initLogger(level, format string) {
 }
 
 // Invoke handlers, services and repositories here
-func initHandlers(e *echo.Echo) {
+func initHandlers(e *echo.Echo, c config.Config) {
 	// App handler setup
 	appsPostgreSQLRepository := apps.NewPostgreSQLRepository()
 	appsService := apps.NewService(appsPostgreSQLRepository)
 	appsHandler := apps.NewHTTPHandler(e, appsService)
 
-	// Define /app routes
-	e.GET("/apps", appsHandler.GetApps)
+	// Setup /app routes
+	apiRoutePrefix := c.ApiRoutePrefix
```

Comment:
the apiRoutePrefix and apiGroup should move above the app handler setup as these will be used for all routes.

Reply:
@davidffrench it will work regardless.